streams/src/main/java/org/apache/kafka/streams/StreamsMetrics.java

```diff
@@ -27,4 +28,17 @@
     Sensor addLatencySensor(String scopeName, String entityName, String operationName, String... tags);
```

Comment:
I wonder if there is a way to use the other sensor calls for the latency sensor too. Or will this one always remain special?